docs/static/offline-plugins.asciidoc

```diff
@@ -60,14 +60,16 @@ To install an offline plugin pack:
 
 . Move the compressed bundle to the machine where you want to install the plugins.
 
-. Run the `bin/logstash-plugin install` subcommand to install the packaged plugins:
+. Run the `bin/logstash-plugin install` subcommand and pass in the file URI of
+the offline plugin pack. For example: 
 +
 ["source","sh",subs="attributes"]
 -------------------------------------------------------------------------------
-bin/logstash-plugin install file:///path/to/logstash-offline-plugins-{logstash_version}.zip
+bin/logstash-plugin install file:///c:/elastic/install/plugins/logstash-offline-plugins-{logstash_version}.zip
 -------------------------------------------------------------------------------
```

Comment:
can we have both window and linux example? I've answered a few people on discuss and issues for windows I want to make sure we don't run into the inverse situation.